### DISCUSSION TITLE
How to contribute to this project

### DISCUSSION THREAD

**Discussion by vulkomilev** [artyom-beilis/pytorch_dlprim#85] [General]
I want to contribute to this project but I don`t know how and from where to begin . Is there some kind of meeting where I can get assigned some specific tasks and get better understanding of the project ?

**Comment by artyom-beilis** [comment]
Ok... All goes there

1st start from building it, running, testing some networks and getting cases you may found something missing.

For example there are lots of operators that aren't implemented and may work flows not covered.

As author of many FOSS projects - you want to contribute to project - start from using one intensively and thus you'll see what you want to work on.

Amount of work is enormos.
